test/acceptance/bytegroup/go.mod

```diff
@@ -0,0 +1,11 @@
+module bytegroup
```

Comment:
Since we won't actually be releasing any of these packages I think we should make test One Big Module, i.e. move this to test/go.mod so we don't have to deal with a bunch of go.mod/go.sum files.  What do you think?

Reply:
Yes that makes sense, I think we could call it something like generatortests?

Reply:
Sounds good to me.

Reply:
OK, then we're all set. Updated the module name and moved the files. 👍